.github/workflows/backport.yml

```
@@ -1,25 +1,30 @@
-#      )
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport')
+        )
+      )
+    env:
+      PR_NUMBER: ${{ github.event.pull_request.number }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)